extensions/wikia/ImageReview/ImageReviewImageStatus.php

```diff
@@ -108,3 +79,39 @@ function efImageReviewDisplayStatus( ImagePage $imagePage, &$html ) {
 
 	return true;
 }
+
+function fetchReviewHistory( $dbr, $cityId, $pageId ) {
```

Comment:
TBH it is temporary solution for a period when 2 ImageReview will be used. After that this will be replaced by call to IR service